src/app/pages/user-settings/user-settings.page.scss

```diff
@@ -23,3 +23,8 @@ ion-item {
     }
   }
 }
+
+ion-content {
```

Comment:
I stedet for å sette denne cssen rett på ion-content så tror jeg det er bedre å lage en subkomponent med maks bredde. Router-animasjonene ser litt rare ut etter denne endringen.

Reply:
legger til <main> element